src/types/schemas/ChatMessage.ts

```diff
@@ -9,17 +9,25 @@ export default interface ChatMessage {
   /**
    * The text sent by the sender
    */
-  text?: string | undefined
+  content?: string | undefined
 
   /**
    * URL of the meta data if any
    */
   metaData?: string | undefined
 
+  /**
+   * Message date
+   */
   sendAt?: Date
 
   /**
    * reaction linked with message
    */
   reactions?: Array<Reaction> | undefined
+
+  /**
+   * contentType
+   */
+  type?: string
```

Comment:
What will this be used for?

Reply:
To know if the content will only be text, in order to avoid displaying image data as a text message for exemple

Reply:
Ok, so update the doc comment to explain what it is used for.

Reply:
Prefer an enum to a string type